### DISCUSSION TITLE
Confusion between docker images on Docker Hub

### DISCUSSION THREAD

**Discussion by VictorioBerra** [emqx/emqx#11558] [Q&A] [answered]
What is the difference between `eqmx` and `eqmx/eqmx`?

https://hub.docker.com/_/emqx

https://hub.docker.com/r/emqx/emqx

**Comment by id** [comment, marked as answer]
@VictorioBerra the first one is built by docker hub infra via https://github.com/docker-library/official-images/blob/master/library/emqx and https://github.com/emqx/emqx-docker.
The second one is built in emqx/emqx CI from https://github.com/emqx/emqx/blob/master/deploy/docker/Dockerfile.

Technically there shouldn't be any difference in binaries, but there is a little difference in what we do in respective Dockerfiles. Also, the https://hub.docker.com/_/emqx could be lagging a little bit comparing to https://hub.docker.com/r/emqx/emqx as we update that one manually.
